docs/import-assistant.md

```diff
@@ -0,0 +1,67 @@
+ # Import Assistant
+ 
+Run powerful AI-enabled commands to assist with your import script development.
+
+&nbsp;
+### Install
+Add an npm script entry to your Edge Delivery project's `package.json`:
+
+```
+"assistant": "aem-import-helper assistant"
+```
```

Comment:
Is that it?  Feels too easy.  Any installation instructions (if required) like:
`npm install -g aem-import-helper`
or
`npm install aem-import-helper --save-dev`

Reply:
I suggest leavingt this section in the README since it covers how to add the helper commands to a project. This doc should just cover the assistant specific commands.